woocommerce/woocommerce
# [Navigation] Remove feature

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/48997

As a follow up to https://github.com/woocommerce/woocommerce/pull/50190, this PR fully removes the Navigation feature. This includes use of `nav_args` when registering pages.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Load JN and poke around to see if its working.
2. Test an extension that previously hooked into the Navigation and has since removed it ([Bookings](https://github.com/woocommerce/woocommerce-bookings))
3. Test an extension that previously hooked into the Navigation and has NOT removed that code ([Klarna](https://github.com/krokedil/klarna-payments-for-woocommerce))
4. Make sure no fatals exist. Loading the admin page should be sufficient since the code is meant to run on all admin pages.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Remove the Navigation feature

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
